checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -0,0 +1,29 @@
+# Enable StrictMode 3.0
+Set-StrictMode -Version 3.0
+
+class Config {
+    [bool]$rcc
+
+    # Constructor
+    # This is a way to make the rcc property mandatory
+    Config([bool]$rcc) {
+        $this.rcc = $rcc
+    }
+
+    # Method to parse the config file (JSON) and set the value of rcc
+    static [Config] ParseConfigFile($configFilePath) {
+
+        $configFileContent = Get-Content -Raw -Path $configFilePath -ErrorAction Stop
+
+        try {
+            $configData = $configFileContent | ConvertFrom-Json -ErrorAction Stop
```

Comment:
I needed to leave this in the try-catch block because even if the `ConvertFrom-Json` function has an exception the script would continue running. This was the only way, I can think of, to ensure that it will stop with the execution.